server/controllers/finance/patientInvoice.js

```diff
@@ -231,6 +231,11 @@ function find(options) {
     'invoice.uuid IN (SELECT cash_item.invoice_uuid FROM cash_item WHERE cash_item.cash_uuid = ?)'
   );
 
+  filters.custom(
```

Comment:
Hmm... it is probably better to do this filter on the `uuid` itself.  That way it will be super fast (the `uuid` column is INDEXed).

Something like this would be better:
```js
filters.custom('inventory_uuid', 'invoice.uuid IN (SELECT invoice_item.invoice_uuid FROM invoice_item WHERE invoice_item.inventory_uuid = ?');
```